crontab

```diff
@@ -1 +1,2 @@
 * * * * * php ~/www/current/bin/console re:everyMinute >> ~/www/shared/var/log/sms-reminders.log
+* * * * * php ~/www/current/bin/console app:enable-api-client {apiClientName} >> ~/www/shared/var/log/enable-api-client.log
```

Comment:
app:enable-api-client every minutes isn't overkill ?

Reply:
It is